# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/nsIDragSession.idl
# Commit: b24625796fef
# Full Hash: b24625796fefc0ae7b97aa17ef34e072f0b07021
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-07-14 03:18:58
# Description:
#   Bug 1650562: Add missing null check for document BrowsingContext. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D82988
# ==============================================================================

diff -r 432e0c9d2958 -r b24625796fef widget/nsIDragSession.idl
--- a/widget/nsIDragSession.idl	Mon Jul 13 23:11:40 2020 +0300
+++ b/widget/nsIDragSession.idl	Mon Jul 13 18:39:52 2020 +0000
@@ -49,6 +49,7 @@
     * drag originated outside the application. Useful for determining if a drop
     * originated in the same document.
     */
+  [infallible]
   readonly attribute Document sourceDocument;
 
   /**
